src/utils/run-npm-install.js

```diff
@@ -22,6 +22,7 @@ const runNpmInstall = async (workingDirectory, additionalArgs = []) => {
     '--loglevel',
     'error',
     '--no-fund',
+    '--force',
```

Comment:
this is not required but it might reduce errors when installing on npm > 7 due to forced peer dep installs.